recipes/cfgraph/license.md

```diff
@@ -0,0 +1,106 @@
+# Functional Source License, Version 1.1, Apache 2.0 Future License
```

Comment:
Looks like this is the license file upstream: https://github.com/hsolbrig/CFGraph/blob/master/LICENSE